.golangci.goheader.template

```diff
@@ -1,4 +1,4 @@
-Copyright © {{ YEAR }} Meroxa, Inc.
+Copyright © {{ copyright-year }} Meroxa, Inc.
```

Comment:
{{ YEAR }} is the current year, but when a single year is used in a copyright statement, it's actually the start year.